hedera-platform/build.gradle.kts

```diff
@@ -28,7 +28,7 @@ val helidonVersion = "3.2.1"
 val jacksonVersion = "2.13.5"
 val log4jVersion = "2.17.1"
 val mockitoVersion = "4.6.1"
-val nettyVersion = "4.1.87.Final"
+val nettyVersion = "4.1.94.Final"
```

Comment:
GRPC 1.54.1 specifies Netty 4.1.87.Final.
We should probably keep those in sync (either use a newer GRPC, or the older netty).

Reply:
I'll update the version of grpc too